charts/pomerium/values.yaml

```diff
@@ -284,3 +284,18 @@ serviceMonitor:
 
 rbac:
   create: true
+
+redis:
+  enabled: false
+  existingSecret: pomerium-redis-password
```

Comment:
Subchart values cannot be templated to use the release name with `_helpers.tpl` due to helm limitations.